harmony_regridding_service/regridding_service.py

```diff
@@ -100,8 +107,17 @@ def regrid(
         logger.info(f'processed dimension variables: {dimension_vars}')
         vars_to_process -= dimension_vars
 
+        unprocessable_variables = get_unprocessable_variables(var_info, vars_to_process)
+        if unprocessable_variables:
+            if unprocessable_variables & user_requested_variables:
```

Comment:
Not something I care strongly about, but this might be clearer if you used `uprocessable_variables.intersection(user_requested_variables)`. Same a couple of lines lower down.

(I'm thinking that someone might glance at it and think `&` is an `and` rather than an intersection)